freqtrade/main.py

```diff
@@ -191,12 +191,25 @@ def execute_sell(trade: Trade, limit: float) -> None:
 
     fmt_exp_profit = round(trade.calc_profit_percent(rate=limit) * 100, 2)
     profit_trade = trade.calc_profit(rate=limit)
+    current_rate = exchange.get_ticker(trade.pair, False)['bid']
+    current_profit = trade.calc_profit_percent(current_rate)
 
-    message = '*{exchange}:* Selling [{pair}]({pair_url}) with limit `{limit:.8f}`'.format(
+    message = """*{exchange}:* Selling
+*Current Pair:* [{pair}]({pair_url})
+*Limit:* `{limit}`
+*Amount:* `{amount}`
+*Open Rate:* `{open_rate:.8f}`
+*Current Rate:* `{current_rate:.8f}`
+*Current Profit:* `{current_profit:.2f}%`
```

Comment:
I'd say because you are selling it, we should say "Profit", not "Current profit".